src/components/Charts/ DoughnutChart.tsx

```diff
@@ -0,0 +1,29 @@
+import { Doughnut } from 'react-chartjs-2';
```

Comment:
this  file is names with a space at the beginning
lets not do that please
fix and  remove the space in it

Reply:
- [x] separated the tickets as well